src/main/java/org/checkerframework/languageserver/JavacWrapper.java

```diff
@@ -55,6 +67,11 @@ private JavacWrapper(String[] args) {
         // gson = new GsonBuilder().setPrettyPrinting().create();
     }
 
+    /**
+     * Compile function for compile single java file.
+     *
+     * @param f The String for target Java file
```

Comment:
"The String` is redundant, that's already shown. What is the format of the string? An absolute path?

Reply:
> "The String` is redundant, that's already shown. What is the format of the string? An absolute path?

Thanks, I will rewording it.